src/components/YourCompany/CompanyName/CorrectNameRequest.vue

```diff
@@ -58,19 +58,20 @@ import { Component, Prop, Watch, Emit, Mixins } from 'vue-property-decorator'
 import { Action, Getter } from 'vuex-class'
 
 // Mixins
-import { NameRequestMixin } from '@/mixins'
+import { CommonMixin, NameRequestMixin } from '@/mixins'
 
 // Interfaces & Enums
 import {
   ActionBindingIF,
   NameRequestApplicantIF,
   NameRequestIF,
-  NrCorrectionIF
+  NrCorrectionIF,
+  NrResponseIf
```

Comment:
If -> IF

Or does it make more sense as If? If so then we should change for all interfaces.

Reply:
typo! Will fix, thanks

Reply:
Fixed in next commit 👍